<?xml version="1.0"?>
<launch>
  <node name="search_routine" pkg="qforge_ros" type="search_routine.py"/>
</launch>
